docs/rule/no-invalid-role.md

```diff
@@ -0,0 +1,42 @@
+## no-invalid-role
+
+Use of the presentation role for content which should convey semantic information may prevent the user from understanding that content.
```

Comment:
Since this rule is a bit more generic than checking `role="presentation"`, can you tweak the prose here to discuss what `role` is for instead? Then we can add sub-headers for the common types of errors that the rule checks for (which ATM would mostly be only one thing, but in the future will grow).